free-programming-cheatsheets.md

```diff
@@ -5,12 +5,12 @@
 * [Git](#git)
 * [HTML / CSS](#html--css)
 * [JavaScript](#javascript)
-  * [Vue.js](#vuejs)
+* [Vue.js](#vuejs)
```

Comment:
Keep keep this with two spaces before. The this a nested list in the table of contents.

Notice how Vue is an h4 (####), not an h3 (###).

```suggestion
  * [Vue.js](#vuejs)
```